go/cmd/vtctldclient/command/keyspaces.go

```diff
@@ -96,6 +96,14 @@ Otherwise, the keyspace must be empty (have no shards), or returns an error.`,
 		Args:                  cobra.ExactArgs(2),
 		RunE:                  commandSetKeyspaceServedFrom,
 	}
+	// SetKeyspaceDurabilityPolicy makes a SetKeyspaceDurabilityPolicy gRPC call to a vtcltd.
+	SetKeyspaceDurabilityPolicy = &cobra.Command{
+		Use:                   "SetKeyspaceDurabilityPolicy [--durability_policy=policy_name] <keyspace name>",
+		Short:                 "Changes the durability_policy used by the keyspace specified.",
+		DisableFlagsInUseLine: true,
```

Comment:
I don't know either, I just copied over `commandSetKeyspaceServedFrom`

Reply:
When you run `<command> -h`, without this, you'll see a Use line like:

```
vtctldclient SetKeyspaceDurabilityPolicy [--durability_policy=policy_name] <keyspace name> [flags]
```

This removes that last `[flags]` bit, because we're being explicit in the Use line we provide.

Also, we should use hyphens here, not underscores :)

Reply:
Oh, right. My bad, I'll fix this 👍

Reply:
Fixed